app/scripts/controllers/serviceInstance.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .controller('ServiceInstanceController', function ($scope,
+                                                     $filter,
+                                                     $routeParams,
+                                                     DataService,
+                                                     ProjectsService,
+                                                     ServiceInstancesService) {
+    $scope.alerts = {};
+    //$scope.displayName = null;
+    $scope.projectName = $routeParams.project;
+    $scope.serviceInstance = null;
+    $scope.serviceClass = null;
+    $scope.serviceClasses = null;
+
+    $scope.breadcrumbs = [
+      {
+        title: "Provisioned Services",
+        link: "project/" + $routeParams.project + "/browse/service-instances"
+      }
+    ];
+
+    $scope.deprovision = function() {
+      ServiceInstancesService.deprovision($scope.serviceInstance);
+    };
+
+    var watches = [];
+
+    /*var setDisplayName = function() {
+      if(!$scope.serviceInstance || !$scope.serviceClasses) {
+        return;
+      }
+
+      $scope.displayName = $filter('serviceInstanceDisplayName')($scope.serviceInstance, $scope.serviceClasses);
+    };*/
```

Comment:
We should either add back or remove commented out code before we're ready to merge